src/AppBar/DropdownMenu.tsx

```diff
@@ -23,16 +23,16 @@ import styled from '@emotion/styled';
 import clsx from 'clsx';
 
 const Ul = styled('ul')<{ theme?: any }>`
-  ${({ theme }) => css(theme.typography.paragraph)};
+  ${({ theme }) => css(theme.uikit.typography.paragraph)};
```

Comment:
does the theme system allow for the theme to be disabled -> missing, or overwritten with a custom theme that has an entirely different shape? 
in that case these chains may fail with `cannot find X of undefined`.

`theme?.uikit?.typography?.paragraph` covers that, but could also use having a fallback/default value if it the chain ever fails.

Reply:
I like the idea of a fallback. This is an issue we currently face in our projects so it's not an introduced issue but could be a lovely improvement!

A missing theme will default to a default theme.

As for a different shape TS will yell at you but it could happen. I like having a very defined interface for anything we are passing to uikit. There's a lot of improvement to be made on this. plenty of places where we say we're using a theme color but then actually just import a color, so it's never actually dynamically theme-able to the outside. I will create a ticket with this.